10/README.md

```diff
@@ -0,0 +1,88 @@
+```
+shortname: 10/CBD
+Name: Consolidating BigchainDB 2.0 Design
```

Comment:
This title, `Consolidating BigchainDB 2.0 Design` seems to broad and generic. If I read that title, and nothing else, in a year from now, I won't have any idea what the BEP is about. What are the stand-out features of this BEP?

- Golang
- It uses the "Strangler Application" approach to rewriting the code
- It's more about rewriting the code than changing the design. In some ways the high-level design stays the same, because as a black box, it should _do the same thing_ before and after the rewrite.

An alternate title might be, `A Strangler Application Approach to Rewriting Some Code in Go`

Reply:
The short name would also have to be updated.